src/osd/OSD.cc

```diff
@@ -3870,6 +3870,7 @@ int OSD::init()
   // Override a few options if mclock scheduler is enabled.
   maybe_override_options_for_qos();
   maybe_override_max_osd_capacity_for_qos();
+  maybe_override_cost_for_qos();
```

Comment:
Let's move this to the first position as this is a hard override and let max osd capacity change be the last since it runs the osd bench test.